package.json

```diff
@@ -8,8 +8,9 @@
     "tools/**/*"
   ],
   "scripts": {
-    "build": "lerna run build --stream",
+    "build": "lerna run build --stream && yarn build-blazor-app",
```

Comment:
Why add separately? `Lerna` looks through all the apps and builds the repository.

Reply:
see mega-comment